tests/network/connectivity/test_pod_network.py

```diff
@@ -88,8 +87,29 @@ def test_connectivity_over_pod_network(
     """
     Check connectivity
     """
-    dst_ip = get_ip_from_vm_or_virt_handler_pod(family=ip_stack_version_matrix__module__, vm=pod_net_running_vmb)
-    assert dst_ip, f"Cannot get valid IP address from {pod_net_running_vmb.vmi.name}."
+    verify_vm_connectivity_over_pod_network(
+        ip_family=IPV4_STR,
+        src_vm=pod_net_running_vma,
+        dst_vm=pod_net_running_vmb,
+    )
+
 
-    ping_cmd = f"ping -c 3 {dst_ip}"
-    vm_console_run_commands(vm=pod_net_running_vma, commands=[ping_cmd])
+@pytest.mark.gating
+@pytest.mark.polarion("CNV-11845")
+def test_connectivity_over_pod_ipv6_network(
```

Comment:
nit
```suggestion
def test_connectivity_over_pod_network_ipv6(
```
The change improves readability by keeping "pod network" together as a common term and making "IPv6" a clear qualifier.